pkg/quotes/quotes_test.go

```diff
@@ -38,12 +32,13 @@ func (s *quotesSuite) TestQuoteSchema(c *check.C) {
 	}
 	for _, testCase := range cases {
 		name := QuoteSchema(testCase.schema, testCase.table)
-		c.Assert(name, check.Equals, testCase.expected)
+		require.Equal(t, name, testCase.expected)
```

Comment:
I find the function is [func Equal(t TestingT, expected interface{}, actual interface{}, msgAndArgs ...interface{})](https://pkg.go.dev/github.com/stretchr/testify/require#Equal) actually, better to use `require.Equal(t, testCase.expected, name)
Ditto for other places we use `require.Equal`